src/amo/components/Header/styles.scss

```diff
@@ -165,6 +165,12 @@
 
   @include respond-to(medium) {
     @include margin-start(12px);
+    color: $white;
```

Comment:
Instead of copy pasting the same properties from above links css , lets move it all together with links css properties only. WDYT @bobsilverberg :)

Reply:
noted. I will make all the changes in next pr after @bobsilverberg confirms

Reply:
Yes, we can change things around to reuse those properties. I'll make a suggestion for a change that will work.

Reply:
Here is a diff which describes changes that should make this all work:

```diff
diff --git a/src/amo/components/Header/styles.scss b/src/amo/components/Header/styles.scss
index 3500182b0..25bbddfa7 100644
--- a/src/amo/components/Header/styles.scss
+++ b/src/amo/components/Header/styles.scss
@@ -133,18 +133,27 @@
 .Header-extension-workshop-link {
   display: none;
 
-  @include respond-to(large) {
-    &,
-    &:link {
-      @include margin-start(12px);
+  &,
+  &:link {
+    @include margin-start(12px);
+  }
+}
+
+.Header-authenticate-button,
+.Header-developer-hub-link,
+.Header-extension-workshop-link {
+  &,
+  &:link {
+    color: $white;
+    font-size: $font-size-s;
+    font-weight: normal;
+    padding-top: 4px;
+    text-decoration: none;
+  }
 
-      color: $white;
+
+  @include respond-to(large) {
       display: inline-block;
-      font-size: $font-size-s;
-      font-weight: normal;
-      padding-top: 4px;
-      text-decoration: none;
-    }
   }
 }
 
@@ -165,6 +174,8 @@
 
   @include respond-to(medium) {
     @include margin-start(12px);
+
+    vertical-align: baseline;
   }
 }
```